### ISSUE TITLE
Test the code with Intel Cluster OpenMP

### ISSUE THREAD

**Issue by Weiming-Hu** [Weiming-Hu/AnalogsEnsemble#6]
A promising direction for deploying on Clusters by [Intel Cluster OpenMP](https://software.intel.com/sites/default/files/m/b/5/9/UsersGuide.pdf)

**Comment by lec170** [comment]
Awesome. Did you capture any notes while doing this that you could share?

**Comment by Weiming-Hu** [comment]
Unfortunately, the Cluster OpenMP from Intel is discontinued. Please refer to [here](https://software.intel.com/en-us/articles/cluster-openmp-for-intel-compilers):

>Intel has discontinued availability of Cluster OpenMP. Thank you for your interest in this product.

I tried to reproduce the build today and it looks like the option `-cluster-openmp` is no longer available in the version `16.0.3 20160415`.

```
$ icc --version 
icc (ICC) 16.0.3 20160415
Copyright (C) 1985-2016 Intel Corporation.  All rights reserved.

$ icc -cluster-openmp hi.cpp 
icc: command line error: option '-cluster-openmp' not supported
```
